accounting/config.go

```diff
@@ -131,12 +127,11 @@ func NewOnChainConfig(ctx context.Context, lnd lndclient.LndServices, startTime,
 			return lnd.WalletKit.ListSweeps(ctx)
 		},
 		CommonConfig: CommonConfig{
-			StartTime:   startTime,
-			EndTime:     endTime,
-			DisableFiat: disableFiat,
-			FiatBackend: fiatBackend,
-			Granularity: granularity,
-			Categories:  categories,
+			StartTime:      startTime,
+			EndTime:        endTime,
+			DisableFiat:    disableFiat,
```

Comment:
Do you think it makes sense for a nil price source cfg to imply disable fiat? I'm on the fence, because nilness checks can be a bit ambiguous.

Reply:
hmmm i think not cause priceCfg will never be nil since it will be created with default values if no specific options are provided. Thoughts?

Reply:
i guess, could also force priceCfg to be nil earlier on if fiat is disabled?

Reply:
> i guess, could also force priceCfg to be nil earlier on if fiat is disabled?

yeah that was what I was thinking, `if disableFiat, priceCfg=nil`, but I think it'll just be cleaner to keep the bool + ignore if we're disabled. Just a errant thought :)